src/faq.js

```diff
@@ -86,6 +86,14 @@ export default [
       'README of the reuploaded dataset to specify this change.\n\n' +
       'For any questions or concerns please email Franklin: <ffein@stanford.edu>',
   },
+  {
+    faq:
+      'How can I upload data onto OpenNeuro?',
+    faq_answer:
+      'We offer two options for uploading data onto OpenNeuro. ' +
+      'The first is to upload via the web interface. ' +
+      'The second is to upload via our (command-line utility tool)[cli.openneuro.org)',
```

Comment:
The markdown brackets are not correct here.

Reply:
oops - all set